CDCgov/prime-reportstream
# 6469 Fixed time zone in HL7 conversion

This PR fixed the time zone being changed in MSH-7.  The reason for the change in time zone is that the `Bundle.timestamp` is a Java Date that gets converted to the local system time zone.  We just need to use the date in Provenance that keeps the original time zone.

Test Steps:
1. Build this branch.
2. Run the FHIR to HL7 conversion with the command `./prime fhirdata --input-file src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0001.fhir -s metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml`
3. Verify that MSH-7 has the original time zone from the original HL7 data.

## Changes
- Use Provenance as the source of the timestamp
- Updated the segment spreadsheet with the details of the timestamp conversion

## Checklist

### Testing

### Process

## Linked Issues
- #6469

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
